kura/org.eclipse.kura.wire.component.provider/OSGI-INF/metatype/org.eclipse.kura.wire.CloudPublisher.xml

```diff
@@ -47,6 +47,15 @@
             default=""
             description="If set to a non empty value, the publisher will set the message body to the value of the provided STRING or BYTE_ARRAY metric.">
         </AD>
+        
+        <AD id="remove.body.from.metrics"
+            name="Remove body envelope property from metrics"
+            type="Boolean"
+            cardinality="0"
+            required="true"
+            default="false"
+            description="If Set body from envelope property is not an empty value, the publisher will remove from the message metrics the property added in the message body.">
```

Comment:
Maybe rephrase that, the description it is not so clear to me.

Something like: Set true to remove the metric from the message that is set in the message body.

Reply:
Changed in something more simple: "Set true to remove the metric that is sent as body of the message."